Stack/Opc.Ua.Core/Stack/Tcp/TcpTransportListener.cs

```diff
@@ -136,7 +137,7 @@ protected virtual void Dispose(bool disposing)
             m_serverCertificate = settings.ServerCertificate;
             m_serverCertificateChain = settings.ServerCertificateChain;
 
-            m_bufferManager = new BufferManager("Server", (int)Int32.MaxValue, m_quotas.MaxBufferSize);
+            m_bufferManager = new BufferManager("Server", m_quotas.MaxBufferCount, m_quotas.MaxBufferSize);
```

Comment:
I checked that this instance of BufferManager can be removed...

Reply:
It is passed to the TcpServerChannel and used there through the BufferManager property:

[byte[] buffer = BufferManager.TakeBuffer(SendBufferSize, "OnReverseConnectConnectComplete");](https://github.com/mrsuciu/UA-.NETStandard/blob/cda0c55b754fd9d32b124965e983f98d23f24eb8/Stack/Opc.Ua.Core/Stack/Tcp/TcpServerChannel.cs#L155)